website/docs/d/service_account.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_service_account"
+sidebar_current: "docs-kubernetes-data-source-service-account"
+description: |-
+  A service account provides an identity for processes that run in a Pod.
+---
+
+# kubernetes_service_account
+
+A service account provides an identity for processes that run in a Pod.
```

Comment:
This looks good. It's consistent with what we have in a lot of the docs. Let's just add a little extra to let the users know the difference between a service account and what this data source does:

```suggestion
A service account provides an identity for processes that run in a Pod. This data source reads the service account and makes specific attributes available to Terraform.
```